scripts/install_deps_ubuntu.sh

```diff
@@ -1,4 +1,5 @@
-sudo apt-get install librocksdb-dev libglib2.0-dev qt4-qmake libqt4-dev libboost-system-dev \
+sudo apt-get install librocksdb-dev libglib2.0-dev qt5-qmake qtbase5-dev libboost-system-dev \
+                     libsnappy-dev libbz2-dev liblz4-dev \
```

Comment:
Why we need these libraries?

Reply:
This is required by sc-web to work with rocksdb properly, will move it there.